packages/components/managed-data-table/src/data-table-settings/data-table-settings-panel.styles.js

```diff
@@ -0,0 +1,23 @@
+import styled from '@emotion/styled';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+
+const Container = styled.div`
+  background-color: ${vars.colorNeutral95};
+  border-radius: ${vars.borderRadius6};
+  padding: ${vars.spacingM};
+`;
+
+const HeaderContainer = styled.div`
+  display: flex;
+  flex: 1;
+  justify-content: space-between;
+`;
+
+const DroppableContainer = styled.div`
+  width: 100%;
+  position: relative;
+  max-width: 420px;
```

Comment:
Where does this value of `420px` come from?  Is there an existing equivalent token value?

Reply:
Well this is part of the original design of the component. The closest token is `constraint-l` which is `496px`. I'm not aware of the reason why this constraint was set to this value. Maybe I can check with UX